admin-base/ui.apps/src/main/js/apiImpl.js

```diff
@@ -661,7 +661,7 @@ class PerAdminImpl {
       let data = new FormData()
       data.append('fromTenant', fromName)
       data.append('toTenant', toName)
-      data.append('tenantTitle', tenantTitle)
+      data.append('tenantTitle', new Blob([tenantTitle], {type: 'application/json; charset=utf-8'}))
```

Comment:
There is an existing function `json(data)`

You wanna use that instead (DRY)

like that
```javascript
  data.append('tenantTitle', json(tenantTitle))
```

Reply:
was almost working with that function, added a new blob function to handle this usecase